source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -190,16 +190,31 @@ HTTP Webhook Log Target
       This setting corresponds to the
       :envvar:`MINIO_LOGGER_WEBHOOK_PROXY` environment variable.
 
+   .. mc-conf:: queue_dir
+
+      .. versionadded:: RELEASE.2023-05-18T00-05-36Z
+
+      *Optional*
+
+      Specify the directory path to enable MinIO's persistent event store for undelivered messages, such as ``/home/events``.
+
+      MinIO stores undelivered events in the specified store while the webhook service is offline and replays the stored events when connectivity resumes.
+
+      This setting corresponds to the
+      :envvar:`MINIO_LOGGER_WEBHOOK_QUEUE_DIR` environment variable.
+
    .. mc-conf:: queue_size
 
       *Optional*
 
       An integer value to use for the queue size for logger webhook targets.
+      The default is ``100000`` events.
+
+      Requires specifying :mc-conf:`~logger_webhook.queue_dir`.
```

Comment:
This is a guess: does `queue_dir` require setting `queue_size`?

Reply:
Since the default is ``100000`` my guess is *no*, we'll just use the default.